cmd/query.go

```diff
@@ -136,6 +138,35 @@ $ %s q denom-trace osmosis 9BBA9A1C257E971E38C1422780CE6F0B0686F0A3085E2D61118D9
 	return cmd
 }
 
+func queryIBCDenomHash(a *appState) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "denom-hash chain_id trace",
+		Short: "query the denom hash info from a given denom trace",
+		Args:  withUsage(cobra.ExactArgs(2)),
+		Example: strings.TrimSpace(fmt.Sprintf(`
+$ %s query denom-hash osmosis transfer/channel-0/uatom
```

Comment:
TIL this is the IBC denom trace and than you can take this and convert to the hash. Cool!